### PR TITLE
Fixed link to build docs and corrected ivf_flat_example 

### PR DESCRIPTION
**Pull request by mmccarty**
Fixed link to build docs and corrected ivf_flat_example notebook for handles to resources kwarg change.

### CODE REVIEW

**Review by cjnolet — APPROVED** [review]
Thanks for fixing these!

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ If installing a version that has not yet been released, the `rapidsai` channel c
 mamba install -c conda-forge -c nvidia -c rapidsai-nightly cuvs=24.10
 ```
 
-Please see the [Build and Install Guide](docs/source/build.md) for more information on installing cuVS and building from source.
+Please see the [Build and Install Guide](docs/source/build.rst) for more information on installing cuVS and building from source.
```
Comment (cjnolet) [inline README.md:56]: Does this render properly for you? I figured we would have to link directly to the docs here: https://docs.rapids.ai/api/cuvs/nightly/build/

Comment (mmccarty) [reply README.md:56]: Good idea! I'll update to link directly to the render docs.

**Pull request merged**